src/subpages/list.tsx

```diff
@@ -217,26 +217,11 @@ const BlockList: React.FC<BlockListProps> = ({ blocks, loading }) => {
         return <>{(nowTimestamp - beforeTimestamp) / 1000}</>;
       },
     },
-    {
-      key: 'columnDifficulty',
-      name: 'Difficulty',
-      minWidth: 50,
-      maxWidth: 200,
-      isRowHeader: true,
-      isResizable: true,
-      isSorted: false,
-      isSortedDescending: true,
-      data: 'string',
-      isPadded: true,
-      onRender: ({ difficulty }) => (
-        <>{parseInt(difficulty).toLocaleString()}</>
-      ),
-    },
```

Comment:
@planetarium/libplanet Is it okay if we remove this field? Don't we use this information everyday?

Reply:
We have **Average difficulty in this page.** so in my opinion, maybe we don't need it anymore.